ZeCamp/ZeCamp.xcodeproj/project.pbxproj

```diff
@@ -373,13 +405,22 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 8983662E1F2B4A73007AA57A /* ZeCamp.xcconfig */;
 			buildSettings = {
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = W46A5HMB5C;
+				PROVISIONING_PROFILE = "";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```

Comment:
Remove these.
The CI server tries to add these back, but It shouldn’t. I'm trying to find a way for it.